### ISSUE TITLE
Breadcrumb: no label for "..." button

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#3723]
#### Describe the issue:

Example on https://developer.microsoft.com/en-us/fabric#/components/breadcrumb

![image](https://user-images.githubusercontent.com/11937034/34938303-e030ba84-fa0d-11e7-9843-e0ee7725302f.png)
![image](https://user-images.githubusercontent.com/11937034/34938322-f2de4e4e-fa0d-11e7-8d18-3afcdeaed9be.png)

#### Actual behavior:
The label value is empty.

#### Expected behavior:
The more button should have a label.

**Comment by Jahnp** [comment]
It seems reasonable to provide an aria-label for the overflow. In OneDrive and SharePoint Document Libraries, we use "More folders". I'm wondering if Fabric's overflow should be configurable, or if it should just provide a generic default value like "More items". @betrue-final-final @cschlechty, thoughts?

**Comment by mikewheaton** [comment]
The text in components is always configurable to support localization, right? But maybe we could default it to English text so it's at least accessible for that audience.

**Comment by betrue-final-final** [comment]
More items seems reasonable. More reasonable than nothing.
